packages/node_modules/@node-red/nodes/core/network/21-httprequest.js

```diff
@@ -88,8 +88,13 @@ module.exports = function(RED) {
             if (msg.method && n.method && (n.method === "use")) {
                 method = msg.method.toUpperCase();          // use the msg parameter
             }
+
+            var isHttps = (/^https/.test(url));
```

Comment:
Should this be case-insensitive just in case...?
```suggestion
            var isHttps = (/^https/i.test(url));
```